cmd/nerdctl/container_inspect_linux_test.go

```diff
@@ -156,6 +157,25 @@ func TestContainerInspectContainsLabel(t *testing.T) {
 	assert.Equal(base.T, "bar", lbs["bar"])
 }
 
+func TestContainerInspectContainsInternalLabel(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	t.Parallel()
+	testContainer := testutil.Identifier(t)
+
+	base := testutil.NewBase(t)
+	defer base.Cmd("rm", "-f", testContainer).Run()
+
+	base.Cmd("run", "-d", "--name", testContainer, "--mount", "type=bind,src=/tmp,dst=/app,readonly=false,bind-propagation=rprivate", testutil.NginxAlpineImage).AssertOK()
+	base.EnsureContainerStarted(testContainer)
+	inspect := base.InspectContainer(testContainer)
+	lbs := inspect.Config.Labels
+
+	// TODO: add more internal labels testcases
```

Comment:
do you still have more tests to add?

Reply:
> do you still have more tests to add?

No. 
There were never tests for internal labels, such as “containerd.io/restart.status” and "nerdctl/hostname", that are beyond the scope of this commit.